packages/theme/src/theme/fontFamilies/index.ts

```diff
@@ -1,6 +1,6 @@
-import { Typeface } from '../../types'
+import { FontFamily } from '../../types'
 
-const System: Typeface = {
+const System: FontFamily = {
```

Comment:
I think, in this case, we've only changed the name of the mixin, right? We should also change it's behaviour so that a `FontFamily` is a `string` instead of an `object`. A font family would be some valid value for `style={{ fontFamily: ... }}`. So it could be something like `"Times New Roman", Times, serif"`.

You'll recall from the Shallot huddle a few weeks ago that I had set things up this way because I thought that React Native didn't support things like `fontWeight: 'bold'` so we had to group things like font family, weight, and style in a single `Typeface` object for consistency. It turned out either I was wrong or they've added support since then.

Now the goal is we can replace something like this:
```
typeface: ${getTypeface('Monospace', 'Bold')}
```
With something more like this:
```
fontFamily: ${getFontFamily('Monospace')},
fontWeight: 'bold'
```
Lmk if this makes sense!